Automattic/jetpack
# AI Assistant: do not register plugins when not connected

## Proposed changes:

The AI Assistant editor plugins rely on a working connection to WordPress.com, so we should not see them when in offline mode for example.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a fresh JN site, running Bleeding edge.
* Go to Settings > Jetpack Constants
* Enable Offline mode
* Go to Posts > Add New
    * See the AI Assistant panel in the Jetpack plugin sidebar.
* Now switch to this branch
* Visit the post editor again
    * The AI Assistant panel should be gone.
* Go back to Settings > Jetpack Constants
* Disable Offline mode
* Connect your site to WordPress.com.
* Go to Posts > Add New
    * The AI Assistant panel should be there now.

**You will also want to make sure the feature remains available on WordPress.com Simple sites.**
